dom.html

```diff
@@ -6269,11 +6272,6 @@ <h3 class="no-num no-ref heading settled" id="index-defined-here"><span class="c
   <h3 class="no-num no-ref heading settled" id="index-defined-elsewhere"><span class="content">Terms defined by reference</span><a class="self-link" href="#index-defined-elsewhere"></a></h3>
   <ul class="index">
    <li>
-    <a data-link-type="biblio">[ECMASCRIPT]</a> defines the following terms:
```

Comment:
Yeah, it's a bug on my side. Got it reported yesterday, will be fixing it shortly.